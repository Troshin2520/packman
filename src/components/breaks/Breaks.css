.wall-block {
    width: 4vw;
    height: 4vw;
    background-color: black;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    align-items: center;

}

.wall-stone {
    width: 1.9vw;
    height: 0.9vw;
    margin: 0vw 0.05vw 0vw 0.05vw;
    background-color: red;
}

.wall-stone-half {
    width: 0.95vw;
    height: 0.9vw;
    background-color: red;
}