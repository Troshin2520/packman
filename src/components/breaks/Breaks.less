.wall {
    @stoneColor: rgb(160, 54, 35);

    width: 4rem;
    height: 4rem;
    background-color: black;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    align-items: stretch;

    &-stone {
        width: 1.8rem;
        height: 0.9rem;
        margin: 0 0.1rem 0 0.1rem;
        background-color: @stoneColor;
    }

    &-stone-half {
        width: 0.9rem;
        height: 0.9rem;
        background-color: @stoneColor;
    }
}

