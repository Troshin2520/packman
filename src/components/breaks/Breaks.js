import React, {Component} from 'react';
import './Breaks.less';

class Breaks extends Component {

  constructor(props) {
    super(props);
  }

  render() {
    return (<div className="wall">
      <div className="stone"></div>
      <div className="stone"></div>
      <div className="stone-half"></div>
      <div className="stone"></div>
      <div className="stone-half"></div>
      <div className="stone"></div>
      <div className="stone"></div>
      <div className="stone-half"></div>
      <div className="stone"></div>
      <div className="stone-half"></div>
    </div>);
  }
}

export default Breaks;