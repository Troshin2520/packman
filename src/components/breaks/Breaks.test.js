import React from 'react';
import Renderer from 'react-test-renderer';
import Breaks from './Breaks';

describe('Breaks tests', () => {

  it('Breaks renders without crashing', () => {
    const component = Renderer.create(
      <Breaks />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
