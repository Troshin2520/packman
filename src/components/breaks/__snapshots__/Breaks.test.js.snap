// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breaks tests Breaks renders without crashing 1`] = `
<div
  className="wall"
>
  <div
    className="stone"
  />
  <div
    className="stone"
  />
  <div
    className="stone-half"
  />
  <div
    className="stone"
  />
  <div
    className="stone-half"
  />
  <div
    className="stone"
  />
  <div
    className="stone"
  />
  <div
    className="stone-half"
  />
  <div
    className="stone"
  />
  <div
    className="stone-half"
  />
</div>
`;
